### ISSUE TITLE
a problem when the canvas rotate

### ISSUE THREAD

**Issue by cundong** [iammert/MusicPlayerView#3]
when call canvas.rotate() on onDraw()

The middle button is always shake.

**Comment by iammert** [comment]
@cundong could not obverse that problem on my note 4(my device) and nexus 4(Genymotion). Is shake so visible? And can you tell me which device you are running? 

onDraw method rotates canvas, draws cover and rotate it back to 0 degree to draw button, progress etc. I know it is little tricky but it was not visible to me.

**Comment by iammert** [comment]
@cundong I will find better solution on free time btw.

**Comment by cundong** [comment]
my device are MOTO G(Android 4.4.4) and Nexus(Android 4.4.4)

I recorded a video ，You can look at

https://github.com/cundong/MusicPlayerView/blob/master/art/device-Nexus4.mp4

**Comment by iammert** [comment]
Now i see. I will fix it as soon as possible. Sorry for the bug.

**Comment by cundong** [comment]
I think，perhaps there is a better solution than "roate a canvas".  but i have no good idea now.

**Comment by iammert** [comment]
Exactly, as I said it is little tricky. I didnt think that it will be visible by people. I did not know what is solution but I know that always there is a better solution :) thanks.
